src/App.js

```diff
@@ -9,7 +10,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
```

Comment:
You could use "rest" operator here

Reply:
ye thks! I think u mean spread operator not rest.